<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMember</class>
 <widget class="QDialog" name="AddMember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; text-decoration: underline;&quot;&gt;Add Member&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="memberNLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="memberName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="memberComboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Regular</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Executive</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Type: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="memberID">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Member</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
